### PR TITLE
Add exercise: name-badge / nothingness

### PR DESCRIPTION
**Pull request by SaschaMann**
This exercise serves as introduction to nothingness.

It also combines string manipulation and conditionals in a similar way to the stage-heralding exercise. Since the stage-heralding exercise is relatively complicated for a concept exercise, the repetition works in our favour as the student will already have been exposed to conditional string manipulation and have seen an exemplar solution. This may reduce the time needed to solve stage-heralding, and should hopefully guide students towards a more idiomatic solution.

---

It's pretty much a 1:1 translation of the Elixir name-badge exercise.

### CODE REVIEW

File: concepts/nothingness/about.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Nothingness
+
+`nothing` is a singleton instance of the `Nothing` type.
+It is used when a function has no return value, or when a variable holds no value.
+
+You can test if a value is identical to `nothing` using the `isnothing` function:
+
+```jldoctest
```
Comment (SaschaMann) [inline concepts/nothingness/about.md:8]:
```suggestion
```julia
```

File: exercises/concept/name-badge/.meta/config.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    "contributors": [
+      "cmcaine"
+    ],
+    "authors": [
+      "SaschaMann"
+    ],
+    "forked_from": ["elixir/name-badge"]
+  }
+
```
Comment (SaschaMann) [inline exercises/concept/name-badge/.meta/config.json:23]:
```suggestion


```

File: exercises/concept/name-badge/.docs/introduction.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+## Nothingness
+
+`nothing` is a singleton instance of the `Nothing` type.
+It is used when a function has no return value, or when a variable holds no value.
+
+You can test if a value is identical to `nothing` using the `isnothing` function:
```
Comment (cmcaine) [inline exercises/concept/name-badge/.docs/introduction.md:8]:
```suggestion
You can test if a value is `nothing` using the `isnothing` function:
```

**Review by cmcaine — APPROVED** [review]
I think "is nothing" is clearer than "is identical to nothing" because the meaning of "identical" in Julia is not explained here.

**Comment by SaschaMann** [comment]
Thanks for the review :)

**Pull request merged**